NEWS

```diff
@@ -2,6 +2,166 @@ News
 ====
 
 
+Version 3.1.0 rc1
+-----------------
+
+*(Released Mon, 10 Feb 2025)*
+
+Upgrade notes
+~~~~~~~~~~~~~
+
+Upgrading is supported from 3.0.x versions.
+
+If you are using the KVM hypervisor, you should consider setting the parameter
+`machine_version` to `pc`. If it is unset, Ganeti will auto-detect the machine
```

Comment:
I generally would not recommend setting `machine_version` to `pc`. Instead the admin should chose an explicit machine version (i.e. `pc-i440fx-7.2`). This guaranties safe live migration across qemu upgrades!
Only for the absentminded admin Ganeti defaults to `pc` on new clusters, which represents the _current_ qemu version and is not stable for live migration between different qemu versions. It just prevents a possible breaking switch to `q35`in future qemu versions.